tests/unit/usecases/test_wallet_service.py

```diff
@@ -31,23 +31,35 @@ def test_address_balance(self, wallet_service_gateway):
         wallet_service_gateway.address_balance.assert_called_once_with(addr, token)
 
     def test_address_history(self, wallet_service_gateway):
-        objs = [TxHistoryEntryFactory() for _ in range(fake.random_int(min=1, max=10))]
-        wallet_service_gateway.address_history.return_value = objs
+        return_obj = {
+            "history": [
+                TxHistoryEntryFactory() for _ in range(fake.random_int(min=1, max=10))
```

Comment:
```suggestion
                TxHistoryEntryFactory().to_dict() for _ in range(fake.random_int(min=1, max=10))
```

The gateway makes the `to_dict` conversion so the returned value would be a dict.

Reply:
Thanks, done in 0c8393e3e9035585ac7cf4ccf6df762031b28f89